nextcord/state.py

```diff
@@ -2307,7 +2307,7 @@ def parse_auto_moderation_action_execution(self, data) -> None:
 
     def parse_guild_audit_log_entry_create(self, data) -> None:
         guild = self._get_guild(int(data["guild_id"]))
-        user_id = int(data["user_id"])
+        user_id = int(data["user_id"]) if data["user_id"] is not None else data["user_id"]
```

Comment:
Does this possibly want `if data.get("user_id") is not None` to account for when it's not actually present in the data payload?

Reply:
According to the Discord API Docs, it's always in the payload. Do you know of situations where it's not present?

Reply:
No, but given it's nullable I wouldn't put it past discord to just simply not provide in the future

Reply:
Good point.